### ISSUE TITLE
Disabling orm/pubsub hooks makes sockets to fail

### ISSUE THREAD

**Issue by Vadorequest** [balderdashy/sails#3003]
Here is my `.sailsrc` file:

```json
{
  "hooks": {
    "orm": false,
    "pubsub": false
  }
}
```

I don't use ORM, so I don't need pubsub (at least I tought).

I used to run my app using `node app.js`, and it works fine that way, but if I do `sails lift` then my code fails because `sails.sockets` is undefined. (in the `sails.config.sockets.onConnect`)

By removing the sailsrc config, it works fine.

So I'm wondering why does these hooks have influence over the `sails.sockets` state.
Anyway, I'll reactivate the hooks in the meantime.

sails 0.10.5 and node 0.12.4

**Comment by CWyrtzen** [comment]
Hi @Vadorequest  - have you found any resolution or update for this?

**Comment by Vadorequest** [comment]
I just enabled the hooks by removing them from the sailsrc file...

**Comment by CWyrtzen** [comment]
Does enabling the hooks via removing them from the sailsrc file provide a sufficient solution to the issue that I can close it?

**Comment by Vadorequest** [comment]
Well, it fixed my issue because it's not a problem for me to enable these hooks... But I believe there is still an issue behind it. Disabling the orm shouldn't break the socket so it's just a workaround to me.

**Comment by sailsbot** [comment]
Thanks for posting, @Vadorequest.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by usmanhafeez147** [comment]
I have installed mongoose orm for that I must have disabled pubsub but socket not working now
